packages/frontend/src/server/ipqs.ts

```diff
@@ -0,0 +1,78 @@
+type ApiResponse = {
+  success: boolean
+  message: string
+  fraud_score: number
+  country_code: string
+  region: string
+  city: string
+  ISP: string
+  ASN: number
+  operating_system: string
+  browser: string
+  organization: string
+  is_crawler: boolean
+  timezone: string
+  mobile: boolean
+  host: string
+  proxy: boolean
+  vpn: boolean
+  tor: boolean
+  active_vpn: boolean
+  active_tor: boolean
+  device_brand: string
+  device_model: string
+  recent_abuse: boolean
+  bot_status: boolean
+  connection_type: string
+  abuse_velocity: string
+  zip_code: string
+  latitude: number
+  longitude: number
+  request_id: string
+}
+
+async function CheckUserIP(ip_address: string, user_agent: string, language: string): Promise<ApiResponse | null> {
```

Comment:
i prefer using camelCase for function and variable names
```suggestion
async function checkUserIP(ipAddress: string, userAgent: string, language: string): Promise<ApiResponse | null> {
```

Reply:
also `user_agent` and `language` unused